# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/1745775.html
# Commit: aaa7e1f81404
# Full Hash: aaa7e1f8140455a9f83be8278720598ea3fe4cd8
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2021-12-14 09:42:05
# Description:
#   Bug 1745775 - Fix invalid surface info indexing r=gfx-reviewers,bradwerth
#   
#   Some parts of the visibility pass have been ported to use the
#   picture graph infrastructure (update pass assignment and bounding
#   rect propagation) but the main visibility pass still relies on
# ==============================================================================

diff -r de1608e5f69b -r aaa7e1f81404 gfx/tests/crashtests/1745775.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/crashtests/1745775.html	Tue Dec 14 01:06:24 2021 +0000
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<head>
+  <style>
+      * {
+          backface-visibility: hidden ! important;
+          filter: blur(63pc) ! important;
+          padding-block-end: 87%;
+          transform: matrix3d(-55, 29, 49313, 4, 80, 16, -127, 112, 8, 45, 127, -27757, 116, 134, 123, 79.00237606484764);
+          scroll-padding-block-end: 22142ch;
+      }
+  </style>
+  <script>
+    window.addEventListener("load", async () => {
+      await new Promise(r => setTimeout(r, 20))
+      const label = document.createElement("label")
+      document.documentElement.appendChild(label)
+      label.scrollIntoView({ "behavior": "smooth", "block": "end", "inline": "end" })
+
+      setTimeout('document.documentElement.className = ""', 300);
+    })
+  </script>
+</head>
+</html>
\ No newline at end of file